tests/functional/lib/helpers.js

```diff
@@ -1217,7 +1217,18 @@ define([
           return type('input[type=email]', email).call(this);
         }
       })
-      .then(type('input[type=password]', password))
+      .then(type('#password', password))
+      .then(() => {
+        // vpassword existing depends on being in an Experiment
+        // so if it doesn't exist, no big deal, carry on.
```

Comment:
👍 - great comment! Using the above scheme on adding the ability to force an experiment/experimentGroup, you should be able to add a functional test in [tests/functional/sign_up.js](https://github.com/mozilla/fxa-content-server/blob/9a8369ce6c8aa732d44fc5063be73c4c77050b95/tests/functional/sign_up.js) that forces both and you can test what you need to test.